proto/wtools/abase/l0/l1.test/Set.test.s

```diff
@@ -42,6 +42,223 @@ function dichotomy( test )
 
 }
 
+//
+
+function as( test )
+{
+  asTemplate( { method : 'as' } );
+
+  function asTemplate( env )
+  {
+    test.case = `${__.entity.exportStringSolo( env )}, null`;
+    var got = _.set[ env.method ]( null );
+    var expected = new Set;
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, undefined`;
+    var got = _.set[ env.method ]( undefined );
+    var expected = new Set;
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, no arguments`;
+    var got = _.set[ env.method ]();
+    var expected = new Set;
+    test.identical( got, expected );
+
+    /* */
+
+    test.case = `${__.entity.exportStringSolo( env )}, boolean false`;
+    var got = _.set[ env.method ]( false );
+    var expected = new Set( [ false ] );
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, boolean true`;
+    var got = _.set[ env.method ]( true );
+    var expected = new Set( [ true ] );
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, NaN`;
+    var got = _.set[ env.method ]( NaN );
+    var expected = new Set( [ NaN ] );
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, a Number`;
+    var src = 123;
+    var got = _.set[ env.method ]( src );
+    var expected = new Set( [ src ] );
+    test.identical( got, expected );
+
+    test.case = `${__.entity.exportStringSolo( env )}, a string primitive`;
+    var src = 'string';
+    var got = _.set[ env.method ]( src );
+    var expected = new Set( [ src ] );
+    test.identical( got, expected );
+
+    //String object has a Symbol.iterator property. should we cover this case in countable?
```

Comment:
String object has a Symbol.iterator property. should we cover this case in countable?